packages/psss-server/src/types.ts

```diff
@@ -17,7 +17,7 @@ export interface SessionCookie {
 
 type PsssRequestBody = Record<string, unknown>;
 
-export interface PsssRequest<>extends Request<ParamsDictionary, unknown, PsssRequestBody, Query> {
+export interface PsssRequest extends Request<ParamsDictionary, unknown, PsssRequestBody, Query> {
```

Comment:
The previous syntax was invalid and the new version of typescript was throwing an error at this...